app/main/views/buyers.py

```diff
@@ -533,8 +533,11 @@ def get_context_data(self, **kwargs):
         if brief['status'] not in CLOSED_PUBLISHED_BRIEF_STATUSES:
             abort(404)
 
-        text_type = str if sys.version_info[0] == 3 else unicode
-        filename = inflection.parameterize(text_type(brief['title']))
+        # Filename must be unicode
+        if sys.version_info[0] == 3:
```

Comment:
Don't think we need this anymore at all now we're not supporting python 2.x?

Reply:
If we're happy to drop Py2 here then great!